api/managementpb/mysql.proto

```diff
@@ -83,6 +83,35 @@ message AddMySQLRequest {
   bool skip_connection_check = 30;
 }
 
+// AddNodeRequest adds new node to inventory while adding new service.
+message AddNodeRequest {
```

Comment:
`XXXRequest` and `XXXResponse` would be used only for RPC requests and responses. `AddNodeParams`?